packages/runtime/container-runtime/src/gc/gcConfigs.ts

```diff
@@ -24,8 +24,9 @@ import {
 	runSessionExpiryKey,
 	runSweepKey,
 	stableGCVersion,
-	throwOnTombstoneLoadKey,
+	throwOnTombstoneLoadOverrideKey,
```

Comment:
nit: I am guessing there must be some documentation explaining all these settings, that would need to get updated eventually.

Reply:
Yeah, we have a document explaining all these settings. Will make a note to update it.

Reply:
https://dev.azure.com/fluidframework/internal/_workitems/edit/6196